ansible/roles/rdo-networks/tasks/create-rdo-networks.yml

```diff
@@ -19,7 +19,7 @@
   os_router:
     cloud: rdocloud
     state: present
-    name: "{{ item.key  }}_router"
+    name: "{{ item.key  }}_router1"
```

Comment:
Did you mean to change this? If so, can you explain it in the commit message?

Reply:
No, Thats by mistake. Reverted.